algorithms/kernel/linear_model/linear_model_train_normeq_finalize_impl.i

```diff
@@ -66,6 +68,7 @@ Status FinalizeKernel<algorithmFPType, cpu>::compute(const NumericTable &xtxTabl
 
         if (&xtxTable != &xtxFinalTable)
         {
+            DAAL_ITTNOTIFY_SCOPED_TASK(computeFinalize.copyToOnline);
```

Comment:
This task has the same name as the one on line 82. It`s confusing for me, do you really need to consider these parts of code as a single task? Also, the name **copyToOnline** says nothing to me, could you please name it more clear?

Reply:
fixed